README.md

```diff
@@ -2,13 +2,10 @@
 [![CircleCI](https://img.shields.io/circleci/build/github/stoplightio/spectral/master)](https://circleci.com/gh/stoplightio/spectral) [![NPM Downloads](https://img.shields.io/npm/dw/@stoplight/spectral?color=blue)](https://www.npmjs.com/package/@stoplight/spectral) [![Stoplight Forest](https://img.shields.io/ecologi/trees/stoplightinc)][stoplight_forest]
 
 - **Custom Rulesets**: Create custom rules to lint JSON or YAML objects
-- **Ready-to-use Rulesets**: Validate and lint **OpenAPI v2 & v3** and **AsyncAPI** Documents
-- **JSON Path Support**: Use JSON path to apply rules to specific parts of your objects
-- **Ready-to-use Functions**: Built-in set of functions to help [create custom rules](https://meta.stoplight.io/docs/spectral/docs/guides/4-custom-rulesets.md#adding-rules). Functions include pattern checks, parameter checks, alphabetical ordering, a specified number of characters, provided keys are present in an object, etc.
+- **Ready-to-use Rulesets**: Validate and lint **OpenAPI v2 & v3.x** and **AsyncAPI** Documents
+- **API Style Guides**: Automated API Style Guides using rulesets improve consistency across all your APIs
+- **Ready-to-use Functions**: Built-in set of functions to help [create custom rules](https://meta.stoplight.io/docs/spectral/e5b9616d6d50c-custom-rulesets#adding-rules). Functions include pattern checks, parameter checks, alphabetical ordering, a specified number of characters, provided keys are present in an object, etc.
 - **Custom Functions**: Create custom functions for advanced use cases
-- **JSON Validation**: Validate JSON with [Ajv](https://www.npmjs.com/package/ajv)
-
-![Demo of Spectral linting an OpenAPI document from the CLI](./docs/img/demo.svg)
```

Comment:
Removed because this isn't really accurate and it had a glitch in it anyway. Now you need to make the `.spectral.yml` file it makes it seem like you can just install this and run lint which hasn't been true since v6.0.